### PR TITLE
Add graphite basicauth doc

### PR DESCRIPTION
**Pull request by shuoliu1**
Add graphite basic auth doc

### CODE REVIEW

File: content/docs/2.3/scalers/graphite.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
++++
+title = "Graphite"
+layout = "scaler"
+availability = "v1.0+"
```
Comment (tomkerkhove) [inline content/docs/2.3/scalers/graphite.md:4]:
```suggestion
availability = "v2.3+"
```

File: content/docs/2.3/scalers/graphite.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    query: stats.counters.http.hello-world.request.count.count # Note: query must return a vector/scalar single element response
+    threshold: '100'
+    queryTime: '-10Minutes'
+```
+
+**Parameter list:**
+
+- `serverAddress` - Address of Graphite
+- `metricName` - Metric name to use
+- `query` - Query to run
```
Comment (tomkerkhove) [inline content/docs/2.3/scalers/graphite.md:30]: Can you expand this a bit please? I see the following above for example: "# Note: query must return a vector/scalar single element response"

Comment (tomkerkhove) [reply content/docs/2.3/scalers/graphite.md:30]: Would be nice to have more explanation for the other fields as well

File: content/docs/2.3/scalers/graphite.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  - type: graphite
+    metadata:
+      serverAddress: http://<graphite-host>:9090
+      metricName: request-count
+      threshold: '100'
+      query: stats.counters.http.hello-world.request.count.count
+      queryTime: '-10Minutes'
+```
+
+
```
Comment (tomkerkhove) [inline content/docs/2.3/scalers/graphite.md:56]:
```suggestion

```

File: content/docs/2.3/scalers/graphite.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+Here is an example of a graphite scaler with Basic Authentication, define the `Secret` and `TriggerAuthentication` as follows
```
Comment (tomkerkhove) [inline content/docs/2.3/scalers/graphite.md:57]:
```suggestion
Here is an example of scaling with the Graphite scaler with basic authentication by defining the `Secret` and `TriggerAuthentication` as follows:
```

File: content/docs/2.3/scalers/graphite.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+**Parameter list:**
+
+- `serverAddress` - Address of Graphite
+- `metricName` - Metric name to use
+- `query` - Query to run
+- `threshold` - Value to start scaling for
+- `queryTime` - Query Time to from 
+
+
```
Comment (tomkerkhove) [inline content/docs/2.3/scalers/graphite.md:34]: We typically have a "Authentication Parameters" section describing what is supported/required, please have a look at our other scalers.

**Review by tomkerkhove — CHANGES_REQUESTED** [review]

**Comment by shuoliu1** [comment]
query must graphite point path
Other parameters are similar to other scales

**Comment by shuoliu1** [comment]
base not master reopen

**Pull request closed without merging**